src/main/java/mServer/crawler/sender/br/BrMissedSendungsFolgenTask.java

```diff
@@ -0,0 +1,58 @@
+
+  private final MediathekReader crawler;
+
+  private final int maximumDays;
+  
+  private BrIdsDTO missedSendungsFolgen;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "missedSendungsFolgen" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)